library/src/main/java/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -48,7 +48,8 @@ internal class HttpTransaction(
     @ColumnInfo(name = "responseHeaders") var responseHeaders: String?,
     @ColumnInfo(name = "responseBody") var responseBody: String?,
     @ColumnInfo(name = "isResponseBodyPlainText") var isResponseBodyPlainText: Boolean = true,
-    @ColumnInfo(name = "responseImageData") var responseImageData: ByteArray?
+    @ColumnInfo(name = "responseImageData") var responseImageData: ByteArray?,
+    @ColumnInfo(name = "isSlowApiCall") var isSlowApiCall: Boolean = false
```

Comment:
You don't need to store this boolean. This can easily be inferred from the `tookMs` field.

Reply:
Since we are not storing the `normalAPIDuration` i believe we need this boolean field.
Or are you suggesting we save `normalAPIDuration` instead of the boolean?